fiboa_cli/convert_utils.py

```diff
@@ -209,9 +214,9 @@ def download_files(self, uris, cache_folder=None):
                     name = name_from_uri(uri)
                     # if there's no file extension, it's likely a folder, which may not be unique
                     if "." not in name:
-                        name = str(i)
```

Comment:
If i is not used anymore, we can remove it above.

Reply:
Valid point...